zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/integration/ITElasticsearchStorage.java

```diff
@@ -163,12 +163,14 @@ static void aggregateDependencies(ElasticsearchStorage storage, List<Span> spans
     final AggregatedHttpResponse response =
       webClient.execute(HttpRequest.of(RequestHeaders.of(HttpMethod.GET,
         "/_migration/deprecations"))).aggregate().join();
-    if (!response.contentAscii().isEmpty()) {
```

Comment:
as of recent times, the body would never be empty. This compares against the literal empty instead of using json parsing.

Reply:
we didn't look at the log warning, and recently we couldn't have because logging was broken! #3709 fixes logging